org.eclipse.winery.repository/src/main/webapp/jsp/servicetemplates/boundarydefinitions/boundarydefinitions.jsp

```diff
@@ -499,7 +491,6 @@
 						contentType: "application/json"
 					}).done(function (result) {
 						vShowSuccess("Sucessfully created interface");
-						updateInterfaces();
```

Comment:
This method used to update the list of interfaces in case a new interface was created, so that the list contains the new interface as well.
Since the list or typeahead feature for the the provided interfaces and operations was removed, the update for the list is not required anymore. Therefore it was removed.